contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Issue Triage Guidelines
+
+## Purpose
+
+Speed up issue management.
+
```

Comment:
Agree but sorry I didn't get your comment/suggestion here, can you please elaborate? Thanks!